lib/pretty_text.rb

```diff
@@ -227,7 +227,7 @@ def self.avatar_img(avatar_template, size)
   end
 
   def self.unescape_emoji(title)
-    return title unless SiteSetting.enable_emoji?
+    return title unless SiteSetting.enable_emoji? && title
```

Comment:
Why add `&& title`? If `title` is blank?, we should bail out immediately, shouldn't we?

Reply:
But its bailing there. We only continue if `enable_emoji?` is true and a `title` is given.

Reply:
Nevermind, it's `unless`, not `if`... 🤯